.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: CI
+
+on:
+  pull_request:
+  push:
+
+env:
+  RAILS_ENV: test
+  DATABASE_ADAPTER: sqlite3
+
+concurrency:
+  group: ${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  ruby:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - '2.7'
+          - '3.0'
+          - '3.1'
```

Comment:
Do we care about versions other than what we run?

Reply:
future? probably
past? probably not